packages/fhir-group-management/src/components/CommodityAddEdit/tests/Form.test.tsx

```diff
@@ -100,32 +100,38 @@ describe('Health care form', () => {
       wrapper.update();
     });
 
-    expect(toJson(wrapper.find('FormItem#id label'))).toMatchSnapshot('id label');
-    expect(toJson(wrapper.find('FormItem#id input'))).toMatchSnapshot('id field');
+    expect(toJson(wrapper.find('.ant-form-item.id label'))).toMatchSnapshot('id label');
+    expect(toJson(wrapper.find('.ant-form-item.id input'))).toMatchSnapshot('id field');
 
-    expect(toJson(wrapper.find('FormItem#identifier label'))).toMatchSnapshot('identifier label');
-    expect(toJson(wrapper.find('FormItem#identifier input'))).toMatchSnapshot('identifier field');
+    expect(toJson(wrapper.find('.ant-form-item.identifier label'))).toMatchSnapshot(
+      'identifier label'
+    );
+    expect(toJson(wrapper.find('.ant-form-item.identifier input'))).toMatchSnapshot(
+      'identifier field'
+    );
 
-    expect(toJson(wrapper.find('FormItem#name label'))).toMatchSnapshot('name label');
-    expect(toJson(wrapper.find('FormItem#name input'))).toMatchSnapshot('name field');
+    expect(toJson(wrapper.find('.ant-form-item.name label'))).toMatchSnapshot('name label');
+    expect(toJson(wrapper.find('.ant-form-item.name input'))).toMatchSnapshot('name field');
```

Comment:
Instead of introducing and abusing the concept of a class why not use the id of the elements like so:
```suggestion
    expect(toJson(wrapper.find('.ant-form-item#name label'))).toMatchSnapshot('name label');
    expect(toJson(wrapper.find('.ant-form-item#name input'))).toMatchSnapshot('name field');
```